RELEASE.md

```diff
@@ -6,23 +6,25 @@
 
 2. Update version in /VERSION
 
-3. Ensure /CHANGELOG.md is updated, versioned and add the current date
+3. Update version in /bin/elixir and /bin/elixir.bat
```

Comment:
Instead of changing all items, can you update number 2 instead to say: Update version in `/VERSION`, `/bin/elixir`, `/bin/elixir.bat`.

Reply:
Will do.

Reply:
I was thinking it might be better to have it as a separate line item - but I don't mind changing it. Coming up shortly.